aioredis/connection.py

```diff
@@ -107,8 +107,8 @@ def __init__(self, reader, writer, *, address, encoding=None, loop=None):
         self._address = address
         self._loop = loop
         self._waiters = deque()
-        self._parser = hiredis.Reader(protocolError=ProtocolError,
-                                      replyError=ReplyError)
+        self._parser = Reader(protocolError=ProtocolError,
```

Comment:
Why not to add a parser argument for this? And this will let you remove ugly try/except at the parser module bottom.